lib/shouldi/having.ex

```diff
@@ -54,7 +54,11 @@ defmodule ShouldI.Having do
         shouldi_path = unquote(var)[:shouldi_having_path] || []
```

Comment:
What happens if you default `var` to something else, for example `var \\ quote(do: context)`?